registry/remote/repository_test.go

```diff
@@ -1690,6 +1680,13 @@ func TestRepository_DiscoverExtensions(t *testing.T) {
 	}
 }
 
+func TestBlobStoreInterface(t *testing.T) {
```

Comment:
We don't need this test since it won't compile if `*blobStore` is not a `registry.BlobStore`.

Reply:
Removed.